src/components/common/CustomFees.tsx

```diff
@@ -117,11 +120,22 @@ const CustomInput = styled('input')`
   }
 `;
 
-export function getUpToTwoDecimals(input: bigint, decimals: number) {
-  const result = (input * 100n) / 10n ** BigInt(decimals);
+const formatGasPrice = (value: bigint, decimals: number): string => {
+  const formatted = formatUnits(value, decimals);
+  const [wholes, fraction] = formatted.split('.');
 
-  return formatUnits(result, 2);
-}
+  // If something has changed and it's not dot-separated, just return the formatted string.
+  if (!wholes || !fraction) {
+    return formatted;
+  }
+
+  // Otherwise, simplify
+  if (fraction === '0') {
+    return wholes;
+  }
+
+  return formatted;
+};
```

Comment:
This fixes these weird issues when the value in `Custom` field went to `0` and you're no longer able to erase it by clicking backspace.

https://github.com/user-attachments/assets/41c05fb4-7d4b-4af5-bee1-fe4176abdeb5

(I try to erase it completely multiple times in the video above)